CONTRIBUTING.md

```diff
@@ -183,6 +183,20 @@ Here are the trigger phrases for individual checks:
 * `/test-rancher-e2e`: Linux IPv4 e2e tests on Rancher clusters.
 * `/test-rancher-conformance`: Linux IPv4 conformance tests on Rancher clusters.
 * `/test-rancher-networkpolicy`: Linux IPv4 networkpolicy tests on Rancher clusters.
+* `/test-kind-ipv6-e2e`: Kind dual stack e2e tests.
```

Comment:
Maybe use following to make it consistent with other trigger phrases explanation? Similarly for other trigger phrases explanation as well.
```suggestion
* `/test-kind-ipv6-e2e`: Linux dual stack e2e tests on Kind cluster
```